numpy/polynomial/hermite.py

```diff
@@ -1606,6 +1607,70 @@ def hermcompanion(c):
     return mat
 
 
+def herminterp(func, deg, x=None, args=(), window=None):
```

Comment:
Again, I'd omit anything except the Chebyshev functions at this point. The corresponding method would be using Gauss-Hermite integration rather than Gauss-Chebyshev.